### PR TITLE
add known issues topic

### PR DESCRIPTION
**Pull request by mairaw**
Add known issues topic for RC2 plus removed duplicate entry from TOC and fixed a broken link in the TOC

I need to reorganize the SDK/CLI area but I'll leave that for a different PR. Just adding the new topic for now.

/cc @blackdwarf

### CODE REVIEW

File: docs/core-concepts/core-sdk/cli/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+**Workaround 2**
+
+Edit your `.zshrc` and/or `.zshprofile` files to add the `/usr/local/share/dotnet` directory to the $PATH. 
+
+## dotnet restore times out on Windows 7 64-bit edition
+
+**Symptom**
+
+If you have VirtualBox and you try to use the CLI tools on a machine with Windows 7 64-bit edition installed, the `dotnet restore` command will be really slow and will eventually time out without restoring your project.
```
Comment (blackdwarf) [inline docs/core-concepts/core-sdk/cli/known-issues.md]: I need to fix this. It is not just Virtual Box, it is pretty much any virtual networking interface.

File: docs/core-concepts/core-sdk/cli/known-issues.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+[CLI Issue #1732](https://github.com/dotnet/cli/issues/1732) is tracking this problem.
+
+**Affects:** `dotnet restore`
+
+**Workaround**
+
+Disable the VirtualBox network interface and run the `dotnet restore` command.   
+
+## Uninstalling/reinstalling the dotnet PKG files on OS X
```
Comment (blackdwarf) [inline docs/core-concepts/core-sdk/cli/known-issues.md]: We should point people to our uninstall script here.

**Comment by blackdwarf** [comment]
Left some minor comments.

File: docs/core-concepts/core-sdk/cli/known-issues.md (hunk 2 above)
Comment (mairaw) [inline docs/core-concepts/core-sdk/cli/known-issues.md]: @blackdwarf isn't that what i've done?

Comment (blackdwarf) [inline docs/core-concepts/core-sdk/cli/known-issues.md]: @mairaw yes you did, sorry, my bad. As we discussed, can we also leave the link to the discussion and have text that says something to the effect of "this is what the script does" or similar?

Comment (mairaw) [inline docs/core-concepts/core-sdk/cli/known-issues.md]: Sure @blackdwarf

**Comment by blackdwarf** [comment]
@mairaw added the OpenSSL addition to the known issues in the CLI repo. You will want to update this document's openSSL section with the same info.

**Comment by richlander** [comment]
This is the wrong location for the known issues, IMO. It's too buried. We should have a Known issues doc for .NET Core all-up, not on a component basis.

**Comment by mairaw** [comment]
That works too @richlander. Do we have a list of known issues all-up?

**Comment by richlander** [comment]
@mairaw Not in the docs, but in the core repo.

https://github.com/dotnet/core/tree/master/release-notes/1.0

Should we just rely on the release notes and known issues in there?

**Comment by mairaw** [comment]
It's up to you guys what you prefer to do. I can just close this too. For the Framework, we usually don't document those in our docs because that is done in the support site, e.g.: https://support.microsoft.com/en-us/kb/3102432

**Comment by richlander** [comment]
Ya, so then let's just link to the release notes and call it good. If there is info here that isn't yet in those release notes, we should fix that.

We should have a single page in the ".NET Core Guide" that links to each version's release notes.

Closing as a result.

**Pull request closed without merging**